.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+# Continuous Integration Workflow to check pushes to master and pull requests:
+# - code style guidelines are followed (make check-style)
+# - tests pass (make test)
+# - code builds (make dist)
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
```

Comment:
Thoughts on sharing GitHub action config like this plugin is doing here? https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/.github/workflows/ci.yml